doc/sphinx-guides/source/_static/installation/files/root/external-tools/twoRavens.json

```diff
@@ -1,9 +1,11 @@
 {
   "displayName": "TwoRavens",
   "description": "A system of interlocking statistical tools for data exploration, analysis, and meta-analysis.",
+  "toolName": "TwoRavens",
   "scope": "file",
-  "type": "explore",
+  "types": "explore",
```

Comment:
Does this work? I think `types` takes an array.

`type` still works for backward compatibility if we want to revert this.

Reply:
good catch, thx. Changed to type in https://github.com/IQSS/dataverse/commit/582f85e95702b04cbebed52a9b97fc29121d824c